public_html/wp-content/plugins/pattern-creator/pattern-creator.php

```diff
@@ -218,3 +220,27 @@ function rest_api_init() {
 	$controller->register_routes();
 }
 add_action( 'rest_api_init', __NAMESPACE__ . '\rest_api_init' );
+
+/**
+ * Add Twenty Twenty-One styles, set the editor background to white, and add the correct layout settings.
+ *
+ * @param array $settings Default editor settings.
+ * @return array Updated settings.
+ */
```

Comment:
This is going to be hard to understand in a few months after a context switch, or for someone new. Is there a way to over-communicate what's happening?

If we change from Twenty Twenty-One to another theme in the future, does this need to be removed?

Reply:
I added some extra documentation - https://github.com/WordPress/pattern-directory/pull/428/commits/d7445b1d0e1766d18308075c1ec69639b858cf29 & https://github.com/WordPress/pattern-directory/pull/428/commits/3b9731f179e6a18b494ba33c3d0405f2b52df998 - hopefully that clears it up a bit.

> If we change from Twenty Twenty-One to another theme in the future, does this need to be removed?

If we want to change the theme used in the previews, it would need to be updated here & in the [theme Iframe component](https://github.com/WordPress/pattern-directory/blob/b73e61da2dcb998182e82b517e3916269d1e3fe2/public_html/wp-content/themes/pattern-directory/src/components/iframe/index.js#L94). We'll always want to pass _some_ styles though, so this wouldn't be removed entirely.

Reply:
Also, the layout styles will probably always be needed, unless the `wporg-patterns` theme itself switches to a block-based theme (I think… if I'm reading the layout logic correctly).